endpoints/kaspad_requests/submit_transaction_request.py

```diff
@@ -0,0 +1,121 @@
+# encoding: utf-8
+import time
+from typing import List
+
+from pydantic import BaseModel
+
+from server import app, kaspad_client
+
+
+class TxOutpoint(BaseModel):
+    transactionId: str
+    index: int
+
+
+class TxInput(BaseModel):
+    previousOutpoint: TxOutpoint
+    signatureScript: str
+    sequence: int
+    sigOpCount: int
+
+
+class TxScriptPublicKey(BaseModel):
+    version: int
+    scriptPublicKey: str
+
+
+class TxOutputVerboseData(BaseModel):
+    scriptPublicKeyType: str
+    scriptPublicKeyAddress: str
+
+
+class TxOutput(BaseModel):
+    amount: int
+    scriptPublicKey: TxScriptPublicKey
+    # verboseData: TxOutputVerboseData | None
+
+
+class TxVerboseData(BaseModel):
+    transactionId: str
+    hash: str
+    mass: int
+    blockHash: str
+    blockTime: int
+
+
+class TxModel(BaseModel):
```

Comment:
I suggest prefixing this with `Submit` -> `SubmitTxModel` so that it won't be confused with the already existing `TxModel` elsewhere. Same comment for `TxInput` and `TxOutput`.

It's a small point to deal with at this time, but it will make it easier in the future to know that these models are distinct.

Reply:
Done